spec/features/admin/admin_adds_categories_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'Logged in admin' do
+  let(:category) { Category.create!(name: "Monster")}
+
+  before(:each) do
+    admin = User.create(email: "admin@example.com",
+                         password: "password",
+                         full_name: "jeff admin",
+                         display_name: "admin",
+                         role: 1)
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user) { admin }
+
+    visit root_path
+    click_on "Edit Items"
+  end
+
+  scenario 'can add new category' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.